spec/features/content_pages_spec.rb

```diff
@@ -11,7 +11,7 @@
 
   class << self
     def files
-      Dir["app/views/content/**/*.md"]
+      Dir["app/views/content/**/[^_]*.md"]
```

Comment:
What's the rationale here? If we are migrating to markdown there'll be no HTML pages left in the content repo and therefore nothing to test?

Reply:
Ah, I introduced [markdown partials](https://github.com/DFE-Digital/get-into-teaching-content/tree/convert-remaining-content-to-markdown/content/steps-to-become-a-teacher/accordion) for the accordion content and they have no front matter so the tests were failing.

Reply:
Oh I see, I missed that it was checking for the underscore/partial. Makes sense now 👍